### ISSUE TITLE
[TEST] RHEL/CentOS arm64 kubernetes client handshake status 500 Internal Server Error

### ISSUE THREAD

**Issue by yangchiu** [longhorn/longhorn#4685]
## What's the test to develop? Please describe

Currently longhorn automated regression test would encounter lots of kubernetes client connection related errors if running on RHEL/CentOS arm64 ec2 instances.

The error would be like [this](https://ci.longhorn.io/job/public/job/master/job/rhel/job/arm64/job/longhorn-tests-rhel-arm64/31/console):
```
test_backing_image.py::test_csi_io_with_backing_image FAILED

=================================== FAILURES ===================================
________________________ test_csi_io_with_backing_image ________________________

configuration = <kubernetes.client.configuration.Configuration object at 0xffffa7ce29a0>
args = ('GET', '[https://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec](https://10.43.0.1/api/v1/namespaces/default/pods/csi-io-test/exec)')
kwargs = {'_preload_content': True, '_request_timeout': None, 'body': None, 'headers': {'Accept': '*/*', 'User-Agent': 'OpenAPI...IqdpRqp6JeHZo0t_KflBO9ScU8mORuGCmaC1gZjliZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ'}, ...}
url = '[https://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False](https://10.43.0.1/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False)'
_request_timeout = None, _preload_content = True, capture_all = True
headers = {'Accept': '*/*', 'User-Agent': 'OpenAPI-Generator/11.0.0/python', 'authorization': 'bearer eyJhbGciOiJSUzI1NiIsImtpZC...xuFbVFIqdpRqp6JeHZo0t_KflBO9ScU8mORuGCmaC1gZjliZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ'}
query_params = [('command', '/bin/sh'), ('command', '-c'), ('command', 'echo -ne mz7i6hcm90h7woq7itm4r7qvui9hsmxqkcwwcdsqlt72zxfll6nt...nlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re > /data/test'), ('stderr', True), ('stdin', False), ('stdout', True), ...]
key = 'tty', value = False
command = 'echo -ne mz7i6hcm90h7woq7itm4r7qvui9hsmxqkcwwcdsqlt72zxfll6nte9nh5d1qbfj9v771gudza96ne6c55jallj27vbmbbzauatnpebacvzvd...tjz0kj1biqhtq8ifle3fitu2vh5tw4t5vl945ltdc0tgqsz606vy94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re > /data/test'

    def websocket_call(configuration, *args, **kwargs):
        """An internal function to be called in api-client when a websocket
        connection is required. args and kwargs are the parameters of
        apiClient.request method."""
    
        url = args[1]
        _request_timeout = kwargs.get("_request_timeout", 60)
        _preload_content = kwargs.get("_preload_content", True)
        capture_all = kwargs.get("capture_all", True)
        headers = kwargs.get("headers")
    
        # Expand command parameter list to indivitual command params
        query_params = []
        for key, value in kwargs.get("query_params", {}):
            if key == 'command' and isinstance(value, list):
                for command in value:
                    query_params.append((key, command))
            else:
                query_params.append((key, value))
    
        if query_params:
            url += '?' + urlencode(query_params)
    
        try:
>           client = WSClient(configuration, get_websocket_url(url), headers, capture_all)

/usr/local/lib/python3.9/site-packages/kubernetes/stream/ws_client.py:296: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <kubernetes.stream.ws_client.WSClient object at 0xffffa778f2b0>
configuration = <kubernetes.client.configuration.Configuration object at 0xffffa7ce29a0>
url = 'wss://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+m...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False'
headers = {'Accept': '*/*', 'User-Agent': 'OpenAPI-Generator/11.0.0/python', 'authorization': 'bearer eyJhbGciOiJSUzI1NiIsImtpZC...xuFbVFIqdpRqp6JeHZo0t_KflBO9ScU8mORuGCmaC1gZjliZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ'}
capture_all = True

    def __init__(self, configuration, url, headers, capture_all):
        """A websocket client with support for channels.
    
            Exec command uses different channels for different streams. for
        example, 0 is stdin, 1 is stdout and 2 is stderr. Some other API calls
        like port forwarding can forward different pods' streams to different
        channels.
        """
        enableTrace(False)
        header = []
        self._connected = False
        self._channels = {}
        if capture_all:
            self._all = StringIO()
        else:
            self._all = _IgnoredIO()
    
        # We just need to pass the Authorization, ignore all the other
        # http headers we get from the generated code
        if headers and 'authorization' in headers:
            header.append("authorization: %s" % headers['authorization'])
    
        if headers and 'sec-websocket-protocol' in headers:
            header.append("sec-websocket-protocol: %s" %
                          headers['sec-websocket-protocol'])
        else:
            header.append("sec-websocket-protocol: v4.channel.k8s.io")
    
        if url.startswith('wss://') and configuration.verify_ssl:
            ssl_opts = {
                'cert_reqs': ssl.CERT_REQUIRED,
                'ca_certs': configuration.ssl_ca_cert or certifi.where(),
            }
            if configuration.assert_hostname is not None:
                ssl_opts['check_hostname'] = configuration.assert_hostname
        else:
            ssl_opts = {'cert_reqs': ssl.CERT_NONE}
    
        if configuration.cert_file:
            ssl_opts['certfile'] = configuration.cert_file
        if configuration.key_file:
            ssl_opts['keyfile'] = configuration.key_file
    
        self.sock = WebSocket(sslopt=ssl_opts, skip_utf8_validation=False)
        if configuration.proxy:
            proxy_url = urlparse(configuration.proxy)
            self.sock.connect(url, header=header, http_proxy_host=proxy_url.hostname, http_proxy_port=proxy_url.port)
        else:
>           self.sock.connect(url, header=header)

/usr/local/lib/python3.9/site-packages/kubernetes/stream/ws_client.py:94: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <websocket._core.WebSocket object at 0xffffa778fd90>
url = 'wss://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+m...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False'
options = {'header': ['authorization: bearer eyJhbGciOiJSUzI1NiIsImtpZCI6Ii1vLWlPaWNzZmpyWVd6cndOQm9fOHIwUUtUNS1Rbmo5TFRLQVlzQ29...iZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ', 'sec-websocket-protocol: v4.channel.k8s.io']}
addrs = ('10.43.0.1', 443, '/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+m...4uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False')

    def connect(self, url, **options):
        """
        Connect to url. url is websocket url scheme.
        ie. ws://host:port/resource
        You can customize using 'options'.
        If you set "header" list object, you can set your own custom header.
    
        >>> ws = WebSocket()
        >>> ws.connect("ws://echo.websocket.org/",
                ...     header=["User-Agent: MyProgram",
                ...             "x-custom: header"])
    
        Parameters
        ----------
        header: list or dict
            Custom http header list or dict.
        cookie: str
            Cookie value.
        origin: str
            Custom origin url.
        connection: str
            Custom connection header value.
            Default value "Upgrade" set in _handshake.py
        suppress_origin: bool
            Suppress outputting origin header.
        host: str
            Custom host header string.
        timeout: int or float
            Socket timeout time. This value is an integer or float.
            If you set None for this value, it means "use default_timeout value"
        http_proxy_host: str
            HTTP proxy host name.
        http_proxy_port: str or int
            HTTP proxy port. Default is 80.
        http_no_proxy: list
            Whitelisted host names that don't use the proxy.
        http_proxy_auth: tuple
            HTTP proxy auth information. Tuple of username and password. Default is None.
        http_proxy_timeout: int or float
            HTTP proxy timeout, default is 60 sec as per python-socks.
        redirect_limit: int
            Number of redirects to follow.
        subprotocols: list
            List of available subprotocols. Default is None.
        socket: socket
            Pre-initialized stream socket.
        """
        self.sock_opt.timeout = options.get('timeout', self.sock_opt.timeout)
        self.sock, addrs = connect(url, self.sock_opt, proxy_info(**options),
                                   options.pop('socket', None))
    
        try:
>           self.handshake_response = handshake(self.sock, url, *addrs, **options)

/usr/local/lib/python3.9/site-packages/websocket/_core.py:250: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>
url = 'wss://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+m...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False'
hostname = '10.43.0.1', port = 443
resource = '/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+mz7i6hcm90h7woq7itm4...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False'
options = {'header': ['authorization: bearer eyJhbGciOiJSUzI1NiIsImtpZCI6Ii1vLWlPaWNzZmpyWVd6cndOQm9fOHIwUUtUNS1Rbmo5TFRLQVlzQ29...iZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ', 'sec-websocket-protocol: v4.channel.k8s.io']}
headers = ['GET /api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+mz7i6hcm90h7woq....43.0.1', 'Origin: https://10.43.0.1/', 'Sec-WebSocket-Key: WmpTYWM7vkfsB7I2jJJ7MQ==', 'Sec-WebSocket-Version: 13', ...]
key = 'WmpTYWM7vkfsB7I2jJJ7MQ=='
header_str = 'GET /api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne+mz7i6hcm90h7woq7...Pv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ\r\nsec-websocket-protocol: v4.channel.k8s.io\r\n\r\n'

    def handshake(sock, url, hostname, port, resource, **options):
        headers, key = _get_handshake_headers(resource, url, hostname, port, options)
    
        header_str = "\r\n".join(headers)
        send(sock, header_str)
        dump("request header", header_str)
    
>       status, resp = _get_resp_headers(sock)

/usr/local/lib/python3.9/site-packages/websocket/_handshake.py:57: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>
success_statuses = (<HTTPStatus.MOVED_PERMANENTLY: 301>, <HTTPStatus.FOUND: 302>, <HTTPStatus.SEE_OTHER: 303>, <HTTPStatus.SWITCHING_PROTOCOLS: 101>)

    def _get_resp_headers(sock, success_statuses=SUCCESS_STATUSES):
        status, resp_headers, status_message = read_headers(sock)
        if status not in success_statuses:
>           raise WebSocketBadStatusException("Handshake status %d %s", status, status_message, resp_headers)
E           websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error

/usr/local/lib/python3.9/site-packages/websocket/_handshake.py:148: WebSocketBadStatusException

During handling of the above exception, another exception occurred:

client = <longhorn.Client object at 0xffffa7ceb310>
core_api = <kubernetes.client.api.core_v1_api.CoreV1Api object at 0xffffa7433130>
csi_pv_backingimage = {'apiVersion': 'v1', 'kind': 'PersistentVolume', 'metadata': {'name': 'longhorn-testvol-7luu3g'}, 'spec': {'accessMode...aleReplicaTimeout': '30'}, 'volumeHandle': 'longhorn-testvol-7luu3g'}, 'persistentVolumeReclaimPolicy': 'Delete', ...}}
pvc_backingimage = {'apiVersion': 'v1', 'kind': 'PersistentVolumeClaim', 'metadata': {'name': 'longhorn-testvol-7luu3g'}, 'spec': {'acces...'requests': {'storage': '32Mi'}}, 'storageClassName': 'longhorn-csi-pv-test', 'volumeName': 'longhorn-testvol-7luu3g'}}
pod_make = <function pod_make.<locals>.make_pod at 0xffffa7776820>

    @pytest.mark.coretest   # NOQA
    @pytest.mark.backing_image  # NOQA
    @pytest.mark.csi  # NOQA
    def test_csi_io_with_backing_image(client, core_api, csi_pv_backingimage, pvc_backingimage, pod_make):  # NOQA
        for bi_url in (BACKING_IMAGE_QCOW2_URL, BACKING_IMAGE_RAW_URL):
            create_backing_image_with_matching_url(
                client, BACKING_IMAGE_NAME, bi_url)
>           csi_io_test(client, core_api,
                        csi_pv_backingimage, pvc_backingimage, pod_make)

test_backing_image.py:356: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test_csi.py:159: in csi_io_test
    write_pod_volume_data(core_api, pod_name, test_data)
common.py:670: in write_pod_volume_data
    return stream(
/usr/local/lib/python3.9/site-packages/kubernetes/stream/stream.py:35: in stream
    return func(*args, **kwargs)
/usr/local/lib/python3.9/site-packages/kubernetes/client/api/core_v1_api.py:841: in connect_get_namespaced_pod_exec
    (data) = self.connect_get_namespaced_pod_exec_with_http_info(name, namespace, **kwargs)  # noqa: E501
/usr/local/lib/python3.9/site-packages/kubernetes/client/api/core_v1_api.py:927: in connect_get_namespaced_pod_exec_with_http_info
    return self.api_client.call_api(
/usr/local/lib/python3.9/site-packages/kubernetes/client/api_client.py:340: in call_api
    return self.__call_api(resource_path, method,
/usr/local/lib/python3.9/site-packages/kubernetes/client/api_client.py:172: in __call_api
    response_data = self.request(
/usr/local/lib/python3.9/site-packages/kubernetes/stream/stream.py:30: in _intercept_request_call
    return ws_client.websocket_call(config, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

configuration = <kubernetes.client.configuration.Configuration object at 0xffffa7ce29a0>
args = ('GET', '[https://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec](https://10.43.0.1/api/v1/namespaces/default/pods/csi-io-test/exec)')
kwargs = {'_preload_content': True, '_request_timeout': None, 'body': None, 'headers': {'Accept': '*/*', 'User-Agent': 'OpenAPI...IqdpRqp6JeHZo0t_KflBO9ScU8mORuGCmaC1gZjliZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ'}, ...}
url = '[https://10.43.0.1:443/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False](https://10.43.0.1/api/v1/namespaces/default/pods/csi-io-test/exec?command=%2Fbin%2Fsh&command=-c&command=echo+-ne...94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re+%3E+%2Fdata%2Ftest&stderr=True&stdin=False&stdout=True&tty=False)'
_request_timeout = None, _preload_content = True, capture_all = True
headers = {'Accept': '*/*', 'User-Agent': 'OpenAPI-Generator/11.0.0/python', 'authorization': 'bearer eyJhbGciOiJSUzI1NiIsImtpZC...xuFbVFIqdpRqp6JeHZo0t_KflBO9ScU8mORuGCmaC1gZjliZ6MsIPv5PAt19UpfIeyh4N7pWHp0HSWZF9ZyzVByQe24gwPOB775cqeEL0ROJL-KH939RQ'}
query_params = [('command', '/bin/sh'), ('command', '-c'), ('command', 'echo -ne mz7i6hcm90h7woq7itm4r7qvui9hsmxqkcwwcdsqlt72zxfll6nt...nlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re > /data/test'), ('stderr', True), ('stdin', False), ('stdout', True), ...]
key = 'tty', value = False
command = 'echo -ne mz7i6hcm90h7woq7itm4r7qvui9hsmxqkcwwcdsqlt72zxfll6nte9nh5d1qbfj9v771gudza96ne6c55jallj27vbmbbzauatnpebacvzvd...tjz0kj1biqhtq8ifle3fitu2vh5tw4t5vl945ltdc0tgqsz606vy94uhnzkhenlvehn4hx2aszn2ftc17ldci31cmycgj3wboeut420re > /data/test'

    def websocket_call(configuration, *args, **kwargs):
        """An internal function to be called in api-client when a websocket
        connection is required. args and kwargs are the parameters of
        apiClient.request method."""
    
        url = args[1]
        _request_timeout = kwargs.get("_request_timeout", 60)
        _preload_content = kwargs.get("_preload_content", True)
        capture_all = kwargs.get("capture_all", True)
        headers = kwargs.get("headers")
    
        # Expand command parameter list to indivitual command params
        query_params = []
        for key, value in kwargs.get("query_params", {}):
            if key == 'command' and isinstance(value, list):
                for command in value:
                    query_params.append((key, command))
            else:
                query_params.append((key, value))
    
        if query_params:
            url += '?' + urlencode(query_params)
    
        try:
            client = WSClient(configuration, get_websocket_url(url), headers, capture_all)
            if not _preload_content:
                return client
            client.run_forever(timeout=_request_timeout)
            return WSResponse('%s' % ''.join(client.read_all()))
        except (Exception, KeyboardInterrupt, SystemExit) as e:
>           raise ApiException(status=0, reason=str(e))
E           kubernetes.client.rest.ApiException: (0)
E           Reason: Handshake status 500 Internal Server Error

/usr/local/lib/python3.9/site-packages/kubernetes/stream/ws_client.py:302: ApiException
```
The above test only runs one test case `test_csi_io_with_backing_image`, but not just this one test case will fail, lots of test cases will fail with the same error. The like of a full regression test is [here](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2118/consoleFull).

This issue is always reproducible using:
CentOS 8.4 or RHEL 8.3 ~ 8.6
arm64 a1.xlarge ec2 instance
k3s v1.23.1+k3s2

Thanks @mantissahz finds out the root cause: 
Turn out it is because there is some issues in the latest version of busybox (released at 2022/10/05) causes the pod created during longhorn test keeps CrashLoopBackOff on rhel/centos arm64, so the kubernetes client cannot connect to the pod since the pod is not ready, and then causes the above error message.

Currently temporarily we fix busybox version to 1.34.0 to avoid this issue. 

Related discussion about the `Reason: Handshake status 500 Internal Server Error`: https://github.com/kubernetes-client/python/issues/739#issuecomment-475163677

**Comment by yangchiu** [comment]
After https://github.com/longhorn/longhorn-tests/pull/1141 + https://github.com/longhorn/longhorn-tests/pull/1143, `test_autosalvage_with_data_locality_enabled` will still fail, but it also happen in other distro (sles) and arch (amd64), and it started happening before this rhel/centos issue appearing, so it could not related to this issue.

Another type of error appeared after above 2 PRs is `OSError: [Errno 8] Exec format error: 'kubectl'` happened in `test_offline_node` and `test_recovery_from_im_deletion` rhel/centos arm64, investigating now.

**Comment by yangchiu** [comment]
pending by v1.3.2-rc2 release testing items

**Comment by yangchiu** [comment]
> https://ci.longhorn.io/job/public/job/master/job/rhel/job/arm64/job/longhorn-tests-rhel-arm64/37/
> 
> After [longhorn/longhorn-tests#1141](https://github.com/longhorn/longhorn-tests/pull/1141) + [longhorn/longhorn-tests#1143](https://github.com/longhorn/longhorn-tests/pull/1143), `test_autosalvage_with_data_locality_enabled` will still fail, but it also happen in other distro (sles) and arch (amd64), and it started happening before this rhel/centos issue appearing, so it could not related to this issue.
> 
> Another type of error appeared after above 2 PRs is `OSError: [Errno 8] Exec format error: 'kubectl'` happened in `test_offline_node` and `test_recovery_from_im_deletion` rhel/centos arm64, investigating now.
> 
> The last type of error appeared after above 2 PRs is `AssertionError: assert 'e9z1op6f7bgf...pf8jap5q62kuo' == '-ne e9z1op6f...8jap5q62kuo\n' - e9z1op6f7bgfzinq1bg76fkezob4mu9stksje652rqyidnpit22pniiibhb4p7gq20l1x2n6z7pi16u0id70v2ipwofw8wzn05eno6ndbp4pjsesmlja59ugt9m7j7jp0meihnexyuy91qvtri7wn3fkxnjcjzj8fb6a3rg3g5r29vs1mwhqv3yqt2afy5ewn7xe487gcs6qg2agl95byqhe2oqe96ejyxoneuaio7d22cf1pde2dl785i6a6rou9w32e8llge7w4qz3qjumdr6ryulcbu1pxd8qe947vqjd23z78tqzy2recszbomnacrw2p2vy0o8btaqhu4itaayppi5on3jbjd7cjlrhnnqwfbxed0n4vto7x0zgbifs4w5ejvvrysvjl4bpqkwpvigqhd1ud64ovhdjzq727z9661xbgjbi7pzfucc1okch111akbenz1yh8aooxjekrb1omfzbuob8c120zo3dheqtkmnb54upf8jap5q62kuo + -ne e9z1op6f7bgfzinq1bg76fkezob4mu9stksje652rqyidnpit22pniii... ...Full output truncated (1 line hidden), use '-vv' to show` happened in `test_rwx_deployment_with_multi_pods` rhel/centos arm64, which isn't like normal assertion failure caused by timeout, but it's just like wrong format when executing the command, not investigated yet.

After PR merged and rerun test on v1.3.x-head, unable to reproduce the above OSError and AssertionError now:
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2183/
https://ci.longhorn.io/job/public/job/master/job/rhel/job/arm64/job/longhorn-tests-rhel-arm64/38/
https://ci.longhorn.io/job/public/job/master/job/centos/job/arm64/job/longhorn-tests-centos-arm64/19/

The only thing remains is the `test_autosalvage_with_data_locality_enabled` failed test case, but its happens in all distro and arch, not related to this ticket. Close this ticket for now.
